src/components/Comments/ReplyCommentItem.js

```diff
@@ -1,26 +1,34 @@
 /* eslint-disable global-require */
+import {useNavigation} from '@react-navigation/native';
 import * as React from 'react';
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 import IconEn from 'react-native-vector-icons/Entypo';
-import {Image, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
-import {useNavigation} from '@react-navigation/native';
 
-import BlockComponent from '../BlockComponent';
-import ButtonHightlight from '../ButtonHighlight';
-import MemoCommentReply from '../../assets/icon/CommentReply';
 import MemoIc_arrow_down_vote_off from '../../assets/arrow/Ic_downvote_off';
-import MemoIc_arrow_upvote_off from '../../assets/arrow/Ic_upvote_off';
 import MemoIc_downvote_on from '../../assets/arrow/Ic_downvote_on';
+import MemoIc_arrow_upvote_off from '../../assets/arrow/Ic_upvote_off';
 import MemoIc_upvote_on from '../../assets/arrow/Ic_upvote_on';
-import useUpdateComment from './hooks/useUpdateComment';
-import {COLORS, FONTS} from '../../utils/theme';
-import {calculateTime} from '../../utils/time';
-import {fonts, normalizeFontSize} from '../../utils/fonts';
-import {getUserId} from '../../utils/users';
+import MemoCommentReply from '../../assets/icon/CommentReply';
+import MemoSendDM from '../../assets/icon/SendDM';
+import SendDMAnonBlock from '../../assets/icons/images/send-dm-anon-black.svg';
+import SendDMBlack from '../../assets/icons/images/send-dm-black.svg';
+import {Context} from '../../context';
+import useDMMessage from '../../hooks/core/chat/useDMMessage';
+import useCreateChat from '../../hooks/screen/useCreateChat';
+import ProfilePicture from '../../screens/ProfileScreen/elements/ProfilePicture';
 import {iVoteComment, voteCommentV2} from '../../service/vote';
 import {removeWhiteSpace} from '../../utils/Utils';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
 import {getCaptionWithLinkStyle} from '../../utils/string/StringUtils';
+import {COLORS, FONTS} from '../../utils/theme';
+import {calculateTime} from '../../utils/time';
+import {getUserId} from '../../utils/users';
+import BlockComponent from '../BlockComponent';
+import BottomSheetMenu from '../BottomSheet/BottomSheetMenu';
+import ButtonHightlight from '../ButtonHighlight';
 import CommentUserName from '../CommentUsername/CommentUsername';
-import ProfilePicture from '../../screens/ProfileScreen/elements/ProfilePicture';
+import useUpdateComment from './hooks/useUpdateComment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been rearranged and new imports have been added. Ensure that all the newly imported modules are being used in the code, otherwise, they should be removed to avoid unnecessary imports which can lead to increased bundle size and slower startup times.



<!-- This is an auto-generated comment by OSS CodeRabbit -->